### PR TITLE
Don't record assets in returnTo property

### PR DESCRIPTION
**Pull request by feedmypixel**
### Problem
Assets were being stored in `locals.returnTo` property

### Solution
This work:
- enables the application to reference client side assets from `/assets` folder
- add `assets` to auth url check so assets are not stored in returnTo locals property

### CODE REVIEW

**Comment by tyom** [comment]
I was trying to avoid us having extra `assets/` directory in public when I removed it earlier.

File: src/middleware/auth.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 module.exports = function auth (req, res, next) {
   const url = req.url
-  const passThrough = /^\/(support|ping|login)\b/.test(url) || req.session.token
+  const passThrough = /^\/(support|ping|login|assets)\b/.test(url) || req.session.token
```
Comment (teneightfive) [inline src/middleware/auth.js]: We also need to add `public` and `favicon.ico` to this exempt list.

File: src/server.js [hunk 2]
```diff
@@ -53,12 +53,9 @@ app.set('view engine', 'njk')
 nunjucks(app, config)
 
 // Static files
-app.use(favicon(path.join(config.root, 'public/images', 'favicon.ico')))
+app.use(favicon(path.join(config.root, 'public/assets/images', 'favicon.ico')))
 app.use(express.static(path.join(config.root, 'public')))
```
Comment (teneightfive) [inline src/server.js]: Might need to set this as `/public` so that it can easily be added to the exempt list?

**Comment by feedmypixel** [comment]
Closing for the moment as the bug is not 100% reproducible

**Pull request closed without merging**